crates/containerd-shim-wasm/src/sandbox/exec.rs

```diff
@@ -145,7 +145,7 @@ pub unsafe fn fork(cgroup: Option<&dyn Cgroup>) -> Result<Context, Error> {
             }
         }
     } else {
-        debug!("no CAP_SYS_ADMIN, not creating new namespaces");
+        debug!("no CAP_SYS_ADMIN, not creating setting cgroup");
```

Comment:
"not creating cgroup"?

Reply:
In this case it would be not setting the cgroup because the cgroup is already created and just passed into this function.

Reply:
then it would be?

```suggestion
        debug!("no CAP_SYS_ADMIN, not setting cgroup");
```

Reply:
lol, I didn't even notice that `creating` was in there after you pointed it out.